frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -73,27 +75,29 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
       case InferenceServiceModelState.UNKNOWN:
         return (
           <Icon
-            data-testid="status-tooltip"
+            data-testid="status-popover"
             role="button"
             aria-label="warning icon"
             status="warning"
             isInline
             tabIndex={0}
             iconSize={iconSize}
+            style={{ cursor: 'pointer' }}
           >
             <OutlinedQuestionCircleIcon />
           </Icon>
         );
       default:
         return (
           <Icon
-            data-testid="status-tooltip"
+            data-testid="status-popover"
```

Comment:
QE doesn't want us to change test ids unless necessary. It's fine to keep the old ids here.